docs/sphinx/install.rst

```diff
@@ -1,128 +1,183 @@
-CUDA Quantum Open Beta Installation
+Getting Started
 *******************************************
 
-Docker Image
---------------------
+This guide walks through how to :ref:`install CUDA Quantum <install-cuda-quantum>` on your system, and how to set up :ref:`VS Code for local development <local-development-with-vscode>`.
+The section on :ref:`connecting to a remote host <connect-to-remote>` contains some
+guidance for application development on a remote host where CUDA Quantum is installed.
 
-Install the Docker Image
-++++++++++++++++++++++++++++++++++++
+.. _install-cuda-quantum:
 
-Docker images for all CUDA Quantum releases are available on the `NGC Container Registry`_.
-To download images from NGC, please follow the following steps if you have not done so already:
+Local Installation
+------------------------------------
 
-- `Create an account <https://ngc.nvidia.com/signin>`__
-- `Sign in <https://ngc.nvidia.com/signin>`__ to access your account and go to `Setup <https://ngc.nvidia.com/setup>`__.
-- Click on `Get API Key` and generate a new key (this will invalidate any existing keys).
-- Follow the instructions that appear to use that key to log in to the NGC registry using Docker.
+A fully featured CUDA Quantum installation including all C++ and Python tools is available as a 
+`Docker image`_. A `Singularity`_ container can easily be created based on these images. Additionally, we distribute pre-built `Python wheels`_ via PyPI.
```

Comment:
I suggest replacing this text with 3 bullet points (with links): Docker, Singularity, Python Wheels.
It will show all options at a glance more clearly.

Reply:
done